docs/guides/integration/docker.md

```diff
@@ -169,3 +169,33 @@ ENV UV_CACHE_DIR=/opt/uv-cache/
 ```
 
 If not mounting the cache, image size can be reduced with `--no-cache` flag.
+
+### Intermediate layers
+
+If you're using uv to manage your project, you can improve build times by moving your transitive
+dependency installation into its own layer via `uv sync --no-locals`.
+
+`uv sync --no-locals` will install all remote dependencies, but ignore any local dependencies,
+including the project itself. Since remote dependencies are typically immutable (whereas local
+dependencies change frequently), installing remote dependencies upfront can be a significant time
+saver.
+
+```dockerfile title="Dockerfile"
+# Install uv
+FROM python:3.12-slim-bullseye
```

Comment:
```suggestion
FROM python:3.12-slim
```

no need to lock in a debian version if we don't use any debian-specific features.